README.md

```diff
@@ -28,7 +28,8 @@ UPSTREAM_CHE_PATH=/path/to/upstream/che
 mvn -DwithoutKeycloak                          `# disable Keycloak support` \
     -Pfast                                     `# skip tests and other verifications` \
     -PmultiThread                              `# enable maven multi threading` \
-    -DlocalCheRepository=${UPSTREAM_CHE_PATH}  `# get already built upstream Che` \
+    -DlocalCheRepository=${UPSTREAM_CHE_PATH}  `# get already built upstream Che local folder` \ 
```

Comment:
> 1. Specifying localCheRepository is still building upstream che in localCheRepository folder

I never experienced this, and don't see, in the code, how it could behave this way. Could you give the precise command you run, in order to trigger the upstream che build in your local upstream Che repository ?

> 2. Disabling the profile checkout-base-che still requires localCheRepository to be set ....

Things are the other way round. Both main use cases are *with or without* `localCheRepository`:
    - when you build *without* `localCheRepository`, it checks out the upstream Che in `target/export` *and* builds it. But  you can remove the `checkout-base-che` profile to *skip the upstream checkout and build* step and only build the RH distribution, using the upstream that was already checked-out and built by a previous build in `target/export` 
    - when you build *with* `localCheRepository`, it *only builds the RH distribution* but, when it needs to access the upstream Che sources (e.g. GWT application or Dashboard for example), or the upstream Che generated artifacts (e.g. assembly-main), it will get them from the local upstream Che repository defined by the `localCheRepository` setting.

> checkout-base-che is not only checking out but also building ....

This profile is automatically selected when the `localCheRepository` property is not provided (the upstream repository should be checked-out when it is not provided). It is only useful when *removing it* to skip the checkout and build (see the previous remark).